TACC/Core-Portal
# task/WP-109-remove-unused-django-fields

## Overview
Many fields connected to the PortalProfile model in the Django backend are no longer used by the portal. Remove all that are no longer needed from the model, and include related migrations.


## Related

* [WP-109](https://jira.tacc.utexas.edu/browse/WP-109)

## Changes
- Removed several fields from PortalProfile model on django backend that were not being used in the CEP anymore (bio, website, orcid_id, etc)
- Removed gender and ethnicity table headers from table on Manage Account page of CEP
- Removed code from the related unit tests that were no longer needed bc fields were removed.
- Removed code in fixtures that was testing for the unnecessary fields, as well.
- Removed the two fields in related migrations file too.


## Testing

1. Log in to CEP
2. Go to your username at the top right and open the dropdown
3. Click on "My Account"
4. View should match table format below. No ethnicity nor gender table header/value should be present
5. Also check https://cep.test/accounts/api/profile/data/
6. There should be no bio, orcid_id, professional_lvel, website, gender, or ethnicity key/value pairs in the api response on that page.

## UI
<img width="649" alt="Screenshot 2023-10-16 at 1 05 24 PM" src="https://github.com/TACC/Core-Portal/assets/50084480/2bd2f27f-27dc-495e-8496-56de3d54ab44">




## Notes
